### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by dougmeredith** [pgina/pgina#384]
It appears there hasn't been a release since 2013 and there hasn't been a commit to the repository since 2016. It appears there was a beta in 2013 that never got released. These aren't generally the signs of a healthy project, but maybe this is just stable, complete, and has no reason to change. 

Educate me. Should I runaway from this project, or jump in with both feet?

**Comment by josh-hemphill** [comment]
Looks like there's a fork that's more active and better documented: https://github.com/MutonUfoAI/pgina

**Comment by dougmeredith** [comment]
Interesting. Thanks, @josh-hemphill.

**Comment by nate-yocom** [comment]
Indeed, to the extent the project is useful to its creator(s), it is dormant.  It is available in its current (historically stable) form in the hopes it proves valuable/useful for those who already use it, or find it useful for themselves.  To the extent it provides fodder for others building Credential Providers etc, it is hopefully a useful reference point (yay open source!).

**Comment by oskarahner** [comment]
@nate-yocom I have a plugin I want to contribute with, is that possible at all ? Its called PasswdPlugin and implements unix-style logins on windows via pGina. Basically You copy the threee files passwd, shadow and group to the windows-host and then You can login to the host.

**Comment by nate-yocom** [comment]
@oskarahner sure! submit a PR and im happy to include it, though I won't build a new release [lack of time/tools/env at this point].  Or, if you'd like, host the src/build of your plugin somewhere (Ala Github) and I'm happy to update the pGina wiki to include links to it (this goes for any/all other plugin authors as well).
